File: hubsmsa-frontend/src/app/shared/shared-models/enum/sexo.enum.ts
export enum SexoEnum {
  MASCULINO = 'M',
  FEMININO = 'F',
  IGNORADO = 'I',
  NAO_SE_APLICA = 'N',
}

export const sexoEnumMensagem = {
  [SexoEnum.MASCULINO]: 'Masculino',
  [SexoEnum.FEMININO]: 'Feminino',
  [SexoEnum.IGNORADO]: 'Ignorado',
  [SexoEnum.NAO_SE_APLICA]: 'Não se aplica'
};

File: hubsmsa-backend/hubsmsa-persistence/src/main/java/br/gov/pbh/prodabel/hubsmsa/enumeration/SexoEnum.java
package br.gov.pbh.prodabel.hubsmsa.enumeration;

/***
 * Enum com o sexo do registro- M - Masculino, F - Feminino ou I - Indefinido
 */
public enum SexoEnum {

  M("3"), F("2"), I("1");

  private String name;

  /**
   * Sexo enum.
   *
   * @param name the name
   */
  private SexoEnum(String name) {
      this.name = name;
  }

  public String getName() {
    return name;
  }

}
